### ISSUE TITLE
Fix CssSyntaxError for //* comments

### ISSUE THREAD

**Issue by XhmikosR** [stylelint/stylelint#3995]
> Clearly describe the bug

Throws `Unknown word` for valid Less code.

> What CSS is needed to reproduce the bug?

```less
//** `<input>` background color
```
> What stylelint configuration is needed to reproduce the bug?

https://github.com/twbs/bootstrap/blob/v3-dev/grunt/.stylelintrc

> Which version of stylelint are you using?

```
C:\Users\xmr\Desktop\bootstrap>node -v && npm -v && npm ls stylelint && npm ls postcss-less
v10.15.3
6.4.1
bootstrap@3.4.1 C:\Users\xmr\Desktop\bootstrap
`-- stylelint@9.10.1

bootstrap@3.4.1 C:\Users\xmr\Desktop\bootstrap
`-- stylelint@9.10.1
  `-- postcss-less@3.1.4
```

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

grunt-stylelint

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Less

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```
Running "stylelint:dist" (stylelint) task

<input css 1>
 51:2  ×  Unknown word   CssSyntaxError
```

This happens in our `v3-dev` branch of `https://github.com/twbs/bootstrap.git`.

I tried a simple test and it seems it's not postcss-less to blame:

```js
const { parse } = require('postcss-less');
const less = '//** `<input>` background color';
const result = parse(less);

console.log(result)
```

returns:

```
Root {
  raws: { semicolon: false, after: '' },
  type: 'root',
  nodes:
   [ Comment {
       raws: [Object],
       type: 'comment',
       parent: [Circular],
       source: [Object],
       inline: true,
       text: '** `<input>` background color' } ],
  source:
   { input:
      Input {
        css: '//** `<input>` background color',
        hasBOM: false,
        id: '<input css 1>' },
     start: { line: 1, column: 1 } } }
```

postcss-less issue: <https://github.com/shellscape/postcss-less/issues/138>

**Comment by TheAifam5** [comment]
I had the same issue and putting space between `//` and `**` fixed the issue.

**Comment by XhmikosR** [comment]
Yeah, but I don't want to change the comments :P

**Comment by TheAifam5** [comment]
@XhmikosR for now I needed to make that change... I was searching for similar issue before I will post my and I found yours.

**Comment by jeddy3** [comment]
@XhmikosR Thanks for the report and for reporting it upstream in the postcss-less.

**Comment by SilencerWeb** [comment]
I have same error on the comment like this: `// Because CellInner has top padding that should be cleared`

Full example:
```jsx
export const Grid = styled.div`
  display: flex;
  padding-bottom: 20px;
  margin-top: -5px; // Because CellInner has top padding that should be cleared
  margin-bottom: -5px; // Because CellInner has bottom padding that should be cleared
`;
```

package.json:
```json
    "stylelint": "^13.0.0",
    "stylelint-config-standard": "^19.0.0",
    "stylelint-config-styled-components": "^0.1.1",
    "stylelint-processor-styled-components": "^1.9.0",
```

.stylelintrc:
```json
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": [
    "stylelint-config-standard",
    "stylelint-config-styled-components"
  ],
  "rules": {
    "declaration-empty-line-before": null
  }
}
```

**Comment by evilebottnawi** [comment]
Do  not use `stylelint-processor-styled-components`, stylelint supported CSS-in-JS out of box

/cc @stylelint/core We have a lot of issues with `stylelint-processor-styled-components`, previously I asked `styled-component` to add a deprecation into README, but no luck, any ideas how we can fix on our side/documentation?

**Comment by XhmikosR** [comment]
BTW I haven't tested if this is still an issue since we don't use Less in >= v4.

**Comment by jeddy3** [comment]
> any ideas how we can fix on our side/documentation?

I've picked this up in https://github.com/stylelint/stylelint/issues/3408.

> BTW I haven't tested if this is still an issue since we don't use Less in >= v4.

I will close this issue as I can't reproduce the [original problem](https://github.com/stylelint/stylelint/issues/3995#issue-424301647) using the demo.

**Comment by mryarbles** [comment]
This is an issue using css-in-js with postcss-lit as the customSyntax override

**Comment by simonkovtyk** [comment]
This problem is still present for .SCSS-Files

**Comment by simonkovtyk** [comment]
Any chances to get this fixed? Or is some PR welcomed?

**Comment by jeddy3** [comment]
@simonkovtyk PRs are welcomed, but please open a [new issue first](https://github.com/stylelint/stylelint/issues/new?template=REPORT_A_BUG.yml) with your reproducible example so that we can best plan a way forward.
